packages/video-intelligence/src/index.js

```diff
@@ -17,56 +17,48 @@
 /*!
  * @module video-intelligence
  */
-
 'use strict';
 
-var v1beta1 = require('./v1beta1/index.js');
+var extend = require('extend');
+var gapic = {
+  v1beta1: require('./v1beta1')
+};
+var gaxGrpc = require('google-gax').grpc();
 
 const VERSION = require('../package.json').version;
 
 /**
- * [Google Cloud Video Intelligence API](https://cloud.google.com/video-intelligence)
- * makes videos searchable, and discoverable, by extracting metadata with an
- * easy to use REST API. You can now search every moment of every video file in
- * your catalog and find every occurrence as well as its significance. It
- * quickly annotates videos stored in Google Cloud Storage, and helps you
- * identify key nouns entities of your video, and when they occur within the
- * video. Separate signal from noise, by retrieving relevant information at the
- * video, shot or per frame.
- *
- * <h2>This is an auto-generated API</h2>
- *
- * It does not follow the conventions you're familiar with from other parts of
- * our library. A handwritten layer is not yet available.
- *
- * The example below shows you how to instantiate the generated client. For
- * further documentation, please browse the
- * [Video Intelligence .proto files](https://github.com/googleapis/googleapis/blob/master/google/cloud/videointelligence/v1beta1/video_intelligence.proto)
- * on GitHub.
+ * Create an videoIntelligenceServiceClient with additional helpers for common
+ * tasks.
  *
- * @constructor
- * @alias module:video-intelligence
+ * Service that implements Google Cloud Video Intelligence API.
```

Comment:
I think it's a matter of taste whether a full sentence or a short fragment is more pleasing to the eye. I'll change it though.